sql-common/net_serv.cc

```diff
@@ -600,8 +600,9 @@ static int begin_packet_write_state(NET *net, uchar command,
     // If we have a payload to compress, then compress_packet will
     // add compression headers for us. This is what we have at this point where
     // each line is an iovec.
-    // | len              |cpn|uncompress len| len                                    | pn | command |
-    // | prefix + command |  0|             0| total_len = command + prefix + payload |  0 |   COM_* |
+    // | len              |cpn|uncompress len| len | pn | command | | prefix +
```

Comment:
The spacing here in this comment is actually important. We should enclose with:
```
/* clang-format off */
/* clang-format on */
```

Reply:
Done